tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -0,0 +1,108 @@
+# stdlib
+import operator
+
+# third party
+import pytest
+import torch
+
+from sympc.session import Session
+from sympc.tensor import MPCTensor
+
+
+def test_mpc_tensor_exception(clients) -> None:
+    alice_client, bob_client = clients
+    # TODO: for more than 2 parties
+    session = Session(parties=[alice_client, bob_client])
+
+    with pytest.raises(ValueError):
+        MPCTensor(secret=42, session=session)
+
+
+def test_reconstruct(clients):
+    alice_client, bob_client = clients
+    # TODO: for more than 2 parties
+    session = Session(parties=[alice_client, bob_client])
+    Session.setup_mpc(session)
+
+    x_secret = torch.Tensor([1, 2, 3, 4])
```

Comment:
MAYBE: Add also some negative values and some float ones.

Reply:
done